packages/story-editor/src/components/canvas/rightClickMenu.js

```diff
@@ -72,6 +111,35 @@ const RightClickMenu = () => {
     };
   }, [ref]);
 
+  const menuButtonRenderer = ({
+    label,
+    shortcut,
+    icon,
+    supportsIcon,
+    SuffixIcon,
+    ...buttonProps
+  }) => (
+    <ContextMenuComponents.MenuButton {...buttonProps}>
+      {supportsIcon && (
+        <LabelWrapper>
+          {icon}
+          {label}
+        </LabelWrapper>
+      )}
+      {!supportsIcon && label}
+      {shortcut && (
+        <ContextMenuComponents.MenuShortcut>
+          {shortcut.display}
+        </ContextMenuComponents.MenuShortcut>
+      )}
+      {SuffixIcon && (
+        <SuffixIconWrapper>
```

Comment:
If the `SuffixWrapper` is necessary then this should exist in `packages/design-system/src/components/contextMenu/components/` to follow the existing pattern.

All context menu components that aren't one-off things should be defined there for re-usability. I believe the `SuffixWrapper` will be one of these things (similar to shortcut) 😄